### PR TITLE
Feature/handle eap6 rule parsing

### PR DESCRIPTION
**Pull request by shawn-hurley**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -181,6 +190,10 @@ func executeRulesets(rulesets []windup.Ruleset, datadir string) (string, string,
 	if err != nil {
 		return "", "", err
 	}
+	if len(converted) == 0 {
+		fmt.Printf("\n\nHERE: %v exit early\n\n", dir)
```
Comment (fabianvf) [inline main.go:194]: probably meant to drop this

File: main.go [hunk 2]
```diff
@@ -165,9 +165,18 @@ func executeRulesets(rulesets []windup.Ruleset, datadir string) (string, string,
 			}
 		}
 
-		err = os.WriteFile(filepath.Join(datadir, JAVA_PROJECT_DIR, "pom.xml"), []byte(POM_XML), 0644)
-		if err != nil {
-			return "", "", err
+		_, err = os.Stat(filepath.Join(datadir, "pom.xml"))
```
Comment (fabianvf) [inline main.go:168]: would you mind adding a comment to explain this?

File: main.go [hunk 3]
```diff
@@ -599,6 +634,12 @@ func convertWindupWhenToAnalyzer(windupWhen windup.When, where map[string]string
 			pattern := strings.Replace(substituteWhere(where, jc.References), "{*}", "*", -1)
 			pattern = strings.Replace(pattern, "(*)", "*", -1)
 			pattern = strings.Replace(pattern, ".*", "*", -1)
+			// Make some .* regesx's more generic.
```
Comment (fabianvf) [inline main.go:637]: hmm, some these change the semantic value of the regex pretty significantly, I'm not sure we want to be doing that

File: main.go [hunk 4]
```diff
@@ -792,7 +833,9 @@ func convertWindupPerformToAnalyzer(perform windup.Iteration, where map[string]s
 		}
 		hint := perform.Hint[0]
 		if hint.Message != "" {
-			ret["message"] = trimMessage(hint.Message)
+			message := trimMessage(hint.Message)
+			// Handle some message replacement
+			ret["message"] = strings.Replace(message, "{package}.{prefix}{type}", "{{name}}", -1)
```
Comment (fabianvf) [inline main.go:838]: :+1:

**Review by fabianvf — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (shawn-hurley) [reply main.go:194]: I did it is in the follow up

Comment (shawn-hurley) [reply main.go]: fixed

File: main.go (hunk 2 above)
Comment (shawn-hurley) [reply main.go:169]: done

File: main.go (hunk 3 above)
Comment (shawn-hurley) [reply main.go:634]: Discussed offline, 

we are choosing to keep this and then to make a todo to come back and deal with capture rules.

**Review by fabianvf — APPROVED** [review]

**Pull request merged**